minify.js

```diff
@@ -156,10 +189,24 @@ for (const file of files) {
   for (const warning of warnings || [])
     console.warn(YELLOW, `[MINIFY][${name}] ${warning}`);
 
-  const code = codeTemplate.replace(
-    /\{\{\s?nginxHost\s?\}\}/gi,
-    '<!--# echo var="http_host" default="" -->'
-  );
+  const code = codeTemplate
+    .replace(
+      /\{\{\s?nginxHost\s?\}\}/gi,
+      '<!--# echo var="http_host" default="" -->'
+    )
+    .replace(
+      /"\{\{\s?overwriteOptions\s?\}\}"/gi,
+      variables.overwriteOptions
+        ? JSON.stringify(variables.overwriteOptions).replace(
+            /:"([^"]+)"/gi,
+            ":$1"
```

Comment:
I wonder if it might not be easier to simply look for the optionally present `INSTALL_OPTIONS` object and its relevant properties inside the script in areas where over-writable options apply, so this intermediary object (and the stringification and string replacing) doesn't need to happen at all. That way the code should remain easier to follow now and later on, especially because this object doesn't actually pass overwriting options on but instead passes references to these (possibly present) values on.